### ISSUE TITLE
Popover border for Menu, ListBox and Picker should have 3:1 contrast against background

### ISSUE THREAD

**Issue by majornista** [adobe/react-spectrum#4045]
# 🐛 Bug Report 

In order to comply with [WCAG 2.2 Success Criterion 1.4.11 Non-text Contrast](https://www.w3.org/TR/WCAG22/#non-text-contrast), the border for interactive controls, like Menus and ListBoxes, should have 3:1 contrast against the background color (the current color has ~2:1 contrast). This means the border color for Popovers, which uses `--spectrum-alias-border-color-dark: var(--spectrum-gray-400);`, needs to be slightly darker (or lighter depending on the theme), `--spectrum-alias-border-color-dark: var(--spectrum-gray-500);`. 

## 🤔 Expected Behavior

The border for interactive controls, like Menus and ListBoxes, should have 3:1 contrast against the background color. This means the border color for Popovers, which uses `--spectrum-alias-border-color-dark: var(--spectrum-gray-400);`, needs to be slightly darker (or lighter depending on the theme), `--spectrum-alias-border-color-dark: var(--spectrum-gray-500);`. 

## 😯 Current Behavior

1. Open [MenuTrigger - isOpen story](https://reactspectrum.blob.core.windows.net/reactspectrum/452507be47323ee045aafd59551ebf0bae48c8f7/storybook/index.html?path=/story/menutrigger--isopen).
2. Using a color contrast checking tool, like [Colour Contrast Analyzer](https://www.tpgi.com/cca-download/), test the Menu's border color, in Light theme `rgb(177, 177, 177)`,  against the background color outside the Menu, `rgb(248, 248, 248)`. 
3. The contrast ratio is currently 2:1.

## 💁 Possible Solution

Change:
https://github.com/adobe/react-spectrum/blob/be0fae9f7952f4d09823498dc1f251e13842f338/packages/%40adobe/spectrum-css-temp/vars/spectrum-global.css#L576

to

```css
  --spectrum-alias-border-color-dark: var(--spectrum-gray-500);
```

## 🔦 Context

Accessibility Audit 

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| react-spectrum   | [@adobe/react-spectrum@3.24.1](https://github.com/adobe/react-spectrum/releases/tag/%40adobe%2Freact-spectrum%403.24.1)
| Browser          | Chrome Version 109.0.5414.119 (Official Build) (arm64)
| Operating System | macOS Ventura 13.2 (22D49)

## 🧢 Your Company/Team

Adobe/Accessibility

## 🕷 Tracking Issue (optional)

A11Y-5848, SDS-12571

**Comment by devongovett** [comment]
We should open this as a ticket against spectrum design rather than here. We can't do anything about it until they update. Also, in the express theme, there is no border at all. I suspect design will have strong opinions on this. Seems debatable that the popover itself is an interactive control.

**Comment by majornista** [comment]
I will open a ticket against Spectrum Design System, but the color variable comes from React-Spectrum:  https://github.com/adobe/react-spectrum/blob/be0fae9f7952f4d09823498dc1f251e13842f338/packages/%40adobe/spectrum-css-temp/vars/spectrum-global.css#L576

I agree that popovers are kind of a gray area with [SC 1.4.11](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html). My feeling is that when the popover contains a menu, listbox, or tree and has a border, the border distinguishes a user interface control from surrounding content, so it should have sufficient contrast.

From: https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html#user-interface-components
> ### Boundaries
>
> This success criterion does not require that controls have a visual boundary indicating the hit area, but if the visual indicator of the control is the only way to identify the control, then that indicator must have sufficient contrast. If text (or an icon) within a button or placeholder text inside a text input is visible and there is no visual indication of the hit area then the Success Criterion is passed. If a button with text also has a colored border, since the border does not provide the only indication there is no contrast requirement beyond the text contrast ([1.4.3 Contrast (Minimum)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum)). Note that for people with cognitive disabilities it is recommended to delineate the boundary of controls to aid in the recognition of controls and therefore the completion of activities.
>
> ![Two buttons, the first with no visual indicator except text saying 'button'. The second is the same but with an added grey border.](https://user-images.githubusercontent.com/154077/218511505-9a30ca0c-3324-4f8a-a4c0-7a0cf8918b3d.png)
> 
> Figure 1 A button (active control) without a visual boundary, and the same button with a focus indicator that is a defined visual boundary of grey (#949494) adjacent to white.

**Comment by majornista** [comment]
Created SDS-12571.

**Comment by devongovett** [comment]
Thanks. Do you think we should add something to that ticket or open another ticket about the Express theme? In that case there is no border at all, and it is kinda difficult to distinguish it from the background. I am worried that this is the direction they are going with Spectrum 2 as well.

<img width="265" alt="image" src="https://user-images.githubusercontent.com/19409/218544000-8b924ba4-112d-43a0-83ff-e65c7f06b2cd.png">

**Comment by LFDanLu** [comment]
Closing for now until we get updated Spectrum designs via SDS-12571.
